--- weechat-4.7.1/src/plugins/ruby/CMakeLists.txt.orig	2025-08-16 21:49:19.000000000 +0200
+++ weechat-4.7.1/src/plugins/ruby/CMakeLists.txt	2025-11-23 12:57:44.889362939 +0100
@@ -19,6 +19,7 @@
 # along with WeeChat.  If not, see <https://www.gnu.org/licenses/>.
 #
 
+add_compile_options("-std=c17")
 add_library(ruby MODULE
   weechat-ruby.c weechat-ruby.h
   weechat-ruby-api.c weechat-ruby-api.h
