--- src/gui/curses/Makefile.am.orig	2010-06-06 13:27:45.458903733 +0200
+++ src/gui/curses/Makefile.am	2010-06-06 13:28:17.417649739 +0200
@@ -25,7 +25,7 @@
                        ../lib_weechat_gui_common.a \
                        ../../core/lib_weechat_core.a \
                        $(PLUGINS_LFLAGS) \
-                       $(NCURSES_LFLAGS) \
+                       $(NCURSES_LFLAGS) -ltinfow \
                        $(GCRYPT_LFLAGS) \
                        $(GNUTLS_LFLAGS)
 
