--- configure.ac.orig	2011-10-22 09:15:47.000000000 +0200
+++ configure.ac	2011-10-23 19:22:22.834371143 +0200
@@ -872,7 +872,7 @@
 enable_eatnewlineglitch="no"
 AC_CACHE_CHECK([for eat_newline_glitch support], ac_have_eatnewlineglitch, [
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM(
-[[ #include <term.h> ]],
+[[ #include <ncursesw/term.h> ]],
 [[ eat_newline_glitch = 0; ]])],
 [ ac_have_eatnewlineglitch="yes" ],
 [ ac_have_eatnewlineglitch="no" ])])
