--- weechat-0.3.1/configure.in	2010-01-23 09:06:36.000000000 +0100
+++ weechat-0.3.1-dud/configure.in	2010-02-18 10:57:44.595581320 +0100
@@ -889,16 +889,6 @@ else
     AC_MSG_RESULT(no)
 fi
 
-CFLAGS=`echo $CFLAGS | sed 's/ -g / /g'`
-CFLAGS=`echo $CFLAGS | sed 's/^-g //g'`
-CFLAGS=`echo $CFLAGS | sed 's/ -g$//g'`
-CFLAGS=`echo $CFLAGS | sed 's/^-g$//g'`
-if test "x$debug" = "x0" ; then
-    CFLAGS="$COMMON_CFLAGS $CFLAGS"
-else
-    CFLAGS="$COMMON_CFLAGS $CFLAGS -g -O0"
-fi
-
 LIBS="$LIBS $INTLLIBS"
 
 case "$host_os" in
