--- weechat-0.3.6/src/plugins/Makefile.am.orig	2011-10-10 18:21:22.000000000 +0200
+++ weechat-0.3.6/src/plugins/Makefile.am	2011-10-23 19:05:30.528472815 +0200
@@ -99,8 +99,8 @@
 # install plugin header
 
 install-data-hook:
-	$(INSTALL) -d $(includedir)/$(PACKAGE)
-	$(INSTALL_DATA) $(abs_top_srcdir)/src/plugins/weechat-plugin.h $(includedir)/$(PACKAGE)
+	$(INSTALL) -d $(DESTDIR)/$(includedir)/$(PACKAGE)
+	$(INSTALL_DATA) $(abs_top_srcdir)/src/plugins/weechat-plugin.h $(DESTDIR)/$(includedir)/$(PACKAGE)
 
 # uninstall plugin header
 
